examples/low_power_lab_receive.rs

```diff
@@ -12,20 +14,19 @@ fn main() -> Result<()> {
     cs.set_direction(Direction::High)?;
 
     // Configure SPI 8 bits, Mode 0
-    let mut spi = Spidev::open("/dev/spidev0.0")?;
+    let mut spi_bus = SpidevBus::open("/dev/spidev0.0")?;
     let options = SpidevOptions::new()
         .bits_per_word(8)
         .max_speed_hz(1_000_000)
         .mode(SpiModeFlags::SPI_MODE_0)
         .build();
-    spi.configure(&options)?;
+    spi_bus.configure(&options)?;
+    let spi_bus = Mutex::new(spi_bus);
+
+    let spi = MutexDevice::new(&spi_bus, cs, Delay);
```

Comment:
I don't think there is a need to complicate things with `Mutex`, it should be enough to use `ExclusiveDevice`.

Reply:
Done.